manifests/nginx.pp

```diff
@@ -80,18 +93,11 @@
       location_custom_cfg_prepend => {
         '    if ($ssl_client_verify != "SUCCESS")' => '{ return 403; } # only allow client-cert authenticated requests',
       },
-      location_cfg_append => $location_cfg_append,
+      proxy_redirect => 'http://localhost/ $scheme://$host:$server_port/',
       notify => Service["nginx"],
     }
   }
 
-  # delete the default.conf to ensure that our virtualhost file gets the requests for localhost.
-  # this is needed if you try and access the server in a browser but not by $hostname_to_use
-  file { "${nginx::params::nx_conf_dir}/conf.d/default.conf":
-    ensure => absent,
-    notify => Service["nginx"],
-  }
-
```

Comment:
they integrated this into the actual puppet module, so we don't have to do it anymore